rholang/src/test/scala/coop/rchain/rholang/interpreter/matcher/MatchTest.scala

```diff
@@ -959,4 +959,21 @@ class VarMatcherSpec extends FlatSpec with Matchers with TimeLimits with TripleE
     } yield (phloLeft.value shouldBe 90)).unsafeRunSync
 
   }
+
+  it should "not finish with a negative cost" in {
```

Comment:
I believe this is a non-requirement. It's very much OK to end with a negative cost account when the last computation costs more than what's left.

Reply:
I'll make it explicit by rewording the test name